src/index.ts

```diff
@@ -94,6 +94,15 @@ export class Transport {
 		}
 	}
 
+	/**
+	 * Closes the channel.
+	 */
+	public async closeChannel() {
```

Comment:
Orizuru uses an interface for Transports (https://github.com/financialforcedev/orizuru/blob/master/src/index.ts) that only knows about close. I don't see why we shouldn't just change the implementation of close, rather than introduce a new method. That way existing code gets the benefit too.

Reply:
Refactor to use only close function with a default parameter for closing the channels